docs/doc/02-cloud/index.md

```diff
@@ -16,9 +16,9 @@ sidebar_label: Databend Cloud
 
 Databend Cloud is now in private beta. To create a Databend Cloud account, go to https://www.databend.com/apply to apply for beta access.
 
-## Logging in to Your Account
+## Logging into Your Account
```

Comment:
corrections for `into` have been fairly controversial. I'm happy to drop them or anything else